packages/office-ui-fabric-react/src/components/List/List.doc.tsx

```diff
@@ -8,37 +8,22 @@ import { ListStatus } from './List.checklist';
 import { createListItems } from '../../utilities/exampleData';
 
 const ListBasicExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/List/examples/List.Basic.Example.tsx') as string;
-const ListGridExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/List/examples/List.Grid.Example.tsx') as string;
-const ListScrollingExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/List/examples/List.Scrolling.Example.tsx') as string;
-const ListGhostingExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/List/examples/List.Ghosting.Example.tsx') as string;
+// const ListGridExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/List/examples/List.Grid.Example.tsx') as string;
```

Comment:
was this just as an experiment? Did you really intend to remove those examples?

Reply:
good catch. not intentional.